charts/uplink/templates/_helpers.tpl

```diff
@@ -31,13 +31,21 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Selector labels
+*/}}
+{{- define "uplink.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "uplink.name" . }}
```

Comment:
Don't forget the unit test (on the deployment) for this change ;)

Reply:
done with https://github.com/jenkins-infra/helm-charts/pull/821/commits/003a84b7e106012ca1cf83255e858d72d772101c